lib/ansible/modules/network/meraki/meraki_organization.py

```diff
@@ -98,6 +98,15 @@
 from ansible.module_utils.network.meraki.meraki import MerakiModule, meraki_argument_spec
 
 
+def get_org(meraki, org_id, data):
+    # meraki.fail_json(msg=str(org_id), data=data, oid0=data[0]['id'], oid1=data[1]['id'])
+    for o in data:
```

Comment:
I have a real dislike for one character variables (even if they are local) because they make it harder to read your code. This could have been **org** instead of **o**.

Reply:
Not a big issue now, but something to think of in the future.
Readability is one of the highest goods in community collaboration.

Reply:
I get it. I got into that habit when I learned C/C++ back in the day as `i` was most often used in a `while` loop. Blame public schools.